packages/office-ui-fabric-react/src/components/TextField/TextField.tsx

```diff
@@ -418,8 +418,10 @@ export class TextField extends BaseComponent<ITextFieldProps, ITextFieldState> i
   }
 
   private _validate(value: string | undefined): void {
+    const { validateOnFocusIn, validateOnFocusOut } = this.props;
+
     // In case of _validate called multi-times during executing validate logic with promise return.
-    if (this._latestValidateValue === value) {
+    if (this._latestValidateValue === value && !(validateOnFocusIn || validateOnFocusOut)) {
       return;
```

Comment:
Can you please describe the problem and the solution in more detail. It is not apparent from the PR exactly what problem you are facing and fixing. Also, could it affect others adversely.

Reply:
@manishgarg1, Thank for comment, added more information in describe